src/mame/namco/mappy.cpp

```diff
@@ -638,71 +638,71 @@ WRITE_LINE_MEMBER(mappy_state::vblank_irq)
+	map(0x8000, 0xffff).rom();  // ROM code (only a000-ffff in Mappy)
 }
 
 void mappy_state::superpac_cpu2_map(address_map &map)
 {
-	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));  /* shared RAM with the main CPU (also sound registers) */
-	map(0x2000, 0x200f).w("mainlatch", FUNC(ls259_device::write_a0));   /* various control bits */
+	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));   // shared RAM with the main CPU (also sound registers)
+	map(0x2000, 0x200f).w("mainlatch", FUNC(ls259_device::write_a0));   // various control bits
 	map(0xe000, 0xffff).rom();
 }
 
 void mappy_state::phozon_cpu2_map(address_map &map)
 {
-	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));  /* shared RAM with the main CPU + sound registers */
-	map(0xe000, 0xffff).rom();                                         /* ROM */
+	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));   // shared RAM with the main CPU + sound registers
+	map(0xe000, 0xffff).rom();  // ROM
 }
 
 void mappy_state::mappy_cpu2_map(address_map &map)
 {
-	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));  /* shared RAM with the main CPU (also sound registers) */
-	map(0x2000, 0x200f).w("mainlatch", FUNC(ls259_device::write_a0));   /* various control bits */
-	map(0xe000, 0xffff).rom();                                         /* ROM code */
+	map(0x0000, 0x03ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));   // shared RAM with the main CPU (also sound registers)
+	map(0x2000, 0x200f).w("mainlatch", FUNC(ls259_device::write_a0));   // various control bits
+	map(0xe000, 0xffff).rom();  // ROM code
 }
 
 
-/* extra CPU only present in Phozon */
+// extra CPU only present in Phozon
 void mappy_state::phozon_cpu3_map(address_map &map)
 {
-	map(0x0000, 0x07ff).ram().w(FUNC(mappy_state::superpac_videoram_w)).share("videoram"); /* video RAM */
-	map(0x0800, 0x1fff).ram().share("spriteram");           /* shared RAM with CPU #2/sprite RAM*/
-	map(0x4000, 0x43ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));  /* shared RAM with CPU #2 */
-	map(0xa000, 0xa7ff).ram();                         /* RAM */
-	map(0xe000, 0xffff).rom();                         /* ROM */
+	map(0x0000, 0x07ff).ram().w(FUNC(mappy_state::superpac_videoram_w)).share("videoram");  // video RAM
+	map(0x0800, 0x1fff).ram().share("spriteram");   // shared RAM with CPU #2/sprite RAM
+	map(0x4000, 0x43ff).rw(m_namco_15xx, FUNC(namco_15xx_device::sharedram_r), FUNC(namco_15xx_device::sharedram_w));   // shared RAM with CPU #2
+	map(0xa000, 0xa7ff).ram();  // RAM
+	map(0xe000, 0xffff).rom();  // ROM
```

Comment:
Smae with these – either line up the comments, or get rid of zero-value comments like these ones.